src/Apps/Artist/Routes/Overview/Components/ArtistEditorialNewsGrid.tsx

```diff
@@ -0,0 +1,271 @@
+import {
+  Skeleton,
+  SkeletonBox,
+  SkeletonText,
+  Image,
+  Text,
+  Column,
+} from "@artsy/palette"
+import {
+  ClickedArticleGroup,
+  ActionType,
+  ContextModule,
+  OwnerType,
+} from "@artsy/cohesion"
+import { GridColumns, ResponsiveBox } from "@artsy/palette"
+import {
+  CellArticleFragmentContainer,
+  CellArticlePlaceholder,
+} from "Components/Cells/CellArticle"
+import { Masonry } from "Components/Masonry"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { RouterLink } from "System/Router/RouterLink"
+import { useSystemContext } from "System/SystemContext"
+import { Media } from "Utils/Responsive"
+import { extractNodes } from "Utils/extractNodes"
+import { take } from "lodash"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useTracking } from "react-tracking"
+import { ArtistEditorialNewsGrid_artist$data } from "__generated__/ArtistEditorialNewsGrid_artist.graphql"
+import { ArtistEditorialNewsGridQuery } from "__generated__/ArtistEditorialNewsGridQuery.graphql"
+import { FC } from "react"
+
+const ARTICLE_COUNT = 6
```

Comment:
If I get it right we grab the first article, then we slice the next 6, but in the query we retrieve 10, so aren't we over fetching data we'd never use?

Reply:
Yep, I had not looked at this — good catch!